app/utils/__init__.py

```diff
@@ -32,10 +32,10 @@
 LOG = utils.log.get_log()
 
 # Build log file names.
-BUILD_LOG_FILE = "build.log"
-BUILD_ERRORS_FILE = "build-errors.log"
-BUILD_WARNINGS_FILE = "build-warnings.log"
-BUILD_MISMATCHES_FILE = "build-mismatches.log"
+BUILD_LOGS_DIR = "logs"
+BUILD_ERRORS_FILE = "logs/build-errors.log"
```

Comment:
Wouldn't it be better to change hardcoded paths to something like 
`BUILD_ERRORS_FILE = os.path.join(BUILD_LOGS_DIR, "build-errors.log")` ?

Reply:
Good point, I shall fix that.